lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
Happy to use either version.

I'm seeing +2% w/ the char codes for Chrome 57.0.2987.133 on OSX 10.11.6.

We could run against a larger corpus and figure out how similar Chrome V8 perf on jsperf.com is too node.js V8 and dump JIT output for [IRHydra2](http://mrale.ph/irhydra/2/), but I doubt there's another 3x win in there.